modules/previousAuctionInfo/index.js

```diff
@@ -80,19 +86,17 @@ export const onAuctionEndHandler = (auctionDetails) => {
               rendered: 0,
               source: 'pbjs',
               adUnitCode: bid.adUnitCode,
-              highestTargetedBidCpm: highestBidsByAdUnitCode[bid.adUnitCode]?.adserverTargeting?.hb_pb || '',
-              targetedBidCpm: receivedBidsMap[bid.bidId]?.adserverTargeting?.hb_pb || '',
               highestBidCpm: highestBidsByAdUnitCode[bid.adUnitCode]?.cpm || 0,
```

Comment:
is 0 better than null? it seems odd that out of 3 cpm fields, one has a different null value than the others. 
Also, I think non-null would be strings, so `"0"` would likely be better than `0`. But `null is best IMO - I don't know if it's true in practice, but in theory the cpm could actually exist and be 0.